tests/scripts/arm32_ci_script.sh

```diff
@@ -229,6 +236,59 @@ function cross_build_coreclr {
     fi
 }
 
+#Cross builds coreclr using Docker
+function cross_build_coreclr_with_docker {
+    __currentWorkingDirectory=`pwd`
+
+    # Check build configuration and choose Docker image
+    if [ "$__buildArch" == "arm" ]; then
+        # TODO: For arm, we are going to embed RootFS inside Docker image.
+        case $__linuxCodeName in
+        trusty)
+            __dockerImage=" chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
```

Comment:
@hqueue, Can you update any new references to chcosta/dotnetcore to use microsoft/dotnet-buildtools-prereqs instead?  I am trying to migrate all of our Docker images to this new repo.  See https://github.com/dotnet/core-setup/pull/1612.  Thanks.